lib/spack/spack/cmd/python.py

```diff
@@ -6,52 +6,54 @@
 from __future__ import print_function
 
 import argparse
-import code
 import os
 import platform
-import runpy
 import sys
 
+import spack_installable.main as sim
+
 import llnl.util.tty as tty
 
 import spack
+import spack.paths
+import spack.util.environment as sue
+import spack.util.executable as exe
```

Comment:
Maybe we can do something like:
```suggestion
from spack.util import environment, executable
```
?